packages/citree/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@jfhbrook/citree",
+  "name": "@matanuska/citree",
```

Comment:
I think I expected to open source this as a generalized component at some point in time. But if I'm only using it in Matanuska and it's "a part of the Matanuska project"...